### ISSUE TITLE
Add CViSB data

### ISSUE THREAD

**Issue by flaneuse** [biothings/metadataplus#9]
## What is the source's name
Center for Viral Systems Biology

## What is the web address of the source
https://data.cvisb.org/dataset/

## Sites to crawl
https://data.cvisb.org/dataset/ebola-viral-seq
https://data.cvisb.org/dataset/lassa-viral-seq
https://data.cvisb.org/dataset/hla
https://data.cvisb.org/dataset/systems-serology

## Does this source require a user account login? Y/N

- [ ] Yes
- [x] No

## In short describe the importance of this source's addition
The mission of the Center for Viral Systems Biology is to identify the immunological, genetic, microbial and physiological factors that play essential roles in determining outcomes from viral infections.
